packages/frontend/src/pages/instructlab/StartInstructLabContainer.svelte

```diff
@@ -0,0 +1,137 @@
+  if (task === undefined) return;
+
+  containerId = task.labels?.['containerId'];
+}
+
+// Submit method when the form is valid
+async function submit(): Promise<void> {
+  errorMsg = undefined;
+  try {
+    trackingId = await instructlabClient.requestCreateInstructlabContainer({
+      connection: $state.snapshot(containerProviderConnection),
+    });
+    router.location.query.set('trackingId', trackingId);
+  } catch (err: unknown) {
+    console.error('Something wrong while trying to create the InstructLab container.', err);
+    errorMsg = String(err);
+  }
+}
+
+// Navigate to the new created service
+function openInstructLabContainer(): void {
+  instructlabClient.routeToInstructLabContainerTerminal(containerId!).catch(console.error);
+}
+</script>
+
+<FormPage title="Run InstructLab as a container">
+  <svelte:fragment slot="content">
+    <div class="flex flex-col w-full">
+      <header>
+        <div class="w-full flex flex-row px-8">
+          {#if available}
+            <Button inProgress={!available} title="Open InstructLab container" on:click={openInstructLabContainer}>
+              Open InstructLab container
+            </Button>
+          {:else}
+            <Button title="Start InstructLab container" inProgress={loading} on:click={submit}>
+              Start InstructLab container
+            </Button>
+          {/if}
+          <Button title="Read documentation" type="link" on:click={submit}>Read documentation</Button>
+        </div>
+      </header>
+      <!-- tasks tracked -->
+      <TrackedTasks class="mx-5 mt-5" onChange={processTasks} trackingId={trackingId} tasks={$tasks} />
+
+      <!-- form -->
+      <div class="bg-[var(--pd-content-card-bg)] m-5 space-y-6 px-8 sm:pb-6 xl:pb-8 rounded-lg h-fit">
+        <div class="w-full">
+          <!-- container provider connection input -->
+          {#if startedContainerProviderConnectionInfo.length > 1}
+            <label for="" class="pt-4 block mb-2 font-bold text-[var(--pd-content-card-header-text)]"
+              >Container engine</label>
+            <ContainerProviderConnectionSelect
+              bind:value={containerProviderConnection}
+              containerProviderConnections={startedContainerProviderConnectionInfo} />
+          {/if}
+
+          <h1 class="pt-4 mb-2 text-lg first-letter:uppercase">Instructions</h1>
+
+          <h2 class="pt-4 mb-2">Create InstructLab configuration</h2>
+
+          <code class="pt-4 mb-2 block">ilab config init</code>
+
+          <h2 class="pt-4 mb-2">Download model</h2>
+
+          <code class="pt-4 mb-2 block">ilab model download</code>
+
+          <h2 class="pt-4 mb-2">Serve the model</h2>
+
+          <code class="pt-4 mb-2 block">ilab model serve</code>
+
+          <h2 class="pt-4 mb-2">Chat with the model</h2>
+
+          <code class="pt-4 mb-2 block">ilab model chat</code>
+
+          {#if errorMsg !== undefined}
+            <ErrorMessage error={errorMsg} />
```

Comment:
error at the bottom ? we have the error included in the task list, so this might not be needed

Reply:
There are cases not covered by tasks. See https://github.com/containers/podman-desktop-extension-ai-lab/blob/cbafb72581c9bcf7a2ea2728ae47b50b50716dbf/packages/frontend/src/pages/CreateService.svelte#L224